src/internal/operators/OperatorSubscriber.ts

```diff
@@ -0,0 +1,48 @@
+/** @prettier */
+import { Subscriber } from '../Subscriber';
+
+export class OperatorSubscriber<T> extends Subscriber<T> {
+  constructor(
+    destination: Subscriber<any>,
+    onNext?: (value: T) => void,
+    onError?: (err: any) => void,
+    onComplete?: () => void,
+    private onUnsubscribe?: () => void
+  ) {
+    super(destination);
+    if (onNext) {
+      this._next = function (value: T) {
+        try {
+          onNext?.(value);
```

Comment:
issue: This `?.` should not be necessary. If, for some reason, TypeScript complains, a non-null assertion - `onNext!(value)` - would be a better option.

Reply:
I think the reason it complains it there's a concern (on the part of the static analyzer, not me) the reference could be reset to undefined before `_next` is called. But you're right, at runtime `!` would be lighter wait.

Reply:
Oh, interestingly it no longer complains. I must have changed something since I added that and not removed it.